content/blog/2023-06-28-etc-transaction-security-what-is-a-good-number-of-block-confirmations/index.md

```diff
@@ -0,0 +1,124 @@
+
+However, that a transaction is accepted on a block and that the uncle block risk is avoided with 7 confirmations, is still not sufficient to consider it safe.
+
+## Avoiding Transaction Reversal Risk
+
+The true risk of a proof of work blockchain is what is called a 51% attack. These attacks are perpetrated by accumulating sufficient hash power to create a longer chain than the one considered the longest one, and then suddenly sending it to the rest of the network which will immediately adopt it as the canonical chain because it is the one with the most work done.
+
+Attackers usually do this to delete a previous transaction they had sent, in effect recovering the money they had sent to a victim. Because the victim thought that they had received the money, they had given the attacker something of value in exchange for the payment. Whenever a 51% attack is done, it usually targets an exchange because the attackers use the fake transaction to send them crypto, then purchase another token and withdraw it, so when they delete the previous transfer they end up with both the money they used to purchase the new token in the first place and the new token.
+
+The way to protect yourself from this kind of attack is to use many confirmations, more specifically, a number that is proportional to the amount of money received.
+
+The logic of this defense mechanism is to use sufficient confirmations to make it economically unfeasible for the attacker to make any money.
+
+This is how it works: If each block has a cost to build, then, when receiving crypto assets, one should wait a number of confirmations that would make the value of that number of blocks equal or greater to the value of the crypto assets received.
+
+For example, if a block in ETC currently costs $38.40 to build, and someone sends you $384.00 in ETC, then if you just wait 10 blocks you would be safe because if any attacker were to spend more capital in making blocks to reverse that transaction they would lose money.
+
+With this logic, we can make these calculations for different transactions values to be safe:
+
+- $1,000 -> wait 26 blocks
+- $5,000 -> wait 130 blocks
+- $10,000 -> wait 260 blocks
+- $100,000 -> wait 2,604 blocks
+- $1,000,000 -> wait 26,041 blocks
+
+## How to Check Transaction Block Confirmations
+
+![Transaction on the ETC blockchain.](./33.png)
+
+As we explained in our previous post, the way to check the statistics of a transaction is to use block explorers. The main block explorer for ETC is [Blockscout](https://blockscout.com/etc/mainnet).
+
+Each transaction on the ETC blockchain has a transaction ID called “transaction hash”. By entering that ID on Blockscout, you can check all its information and in one section at the top is shows the confirmation times with the tag “Confirmed by”.
```

Comment:
Line 98:  Add article; Information added for accuracy; Remove run-on sentence; Correct verb form
```suggestion
Each transaction on the ETC blockchain has a transaction ID called the “transaction hash”. By entering that ID in the search field on Blockscout, you can check all its information. In one section at the top is shown the confirmation times with the tag “Confirmed by”.
```